src/sentry/demo/data_population.py

```diff
@@ -1231,6 +1234,7 @@ def populate_sessions(self, project, distribution_fn_num: int, mobile: bool, err
         seen_versions = []
         num_versions = 0
         weights = []
+        ind_session_threshold = 0.5
```

Comment:
@roggenkemper can we make this a setting on `DEMO_DATA_GEN_PARAMS`?

Reply:
yup!